mqtt/mqtt-broker/src/broker.rs

```diff
@@ -1007,20 +1007,24 @@ pub struct NoSessionError;
 
 #[cfg(test)]
 pub(crate) mod tests {
-    use super::*;
+    use super::{
+        mpsc, proto, Activity, BrokerBuilder, BrokerHandle, BrokerState, ClientEvent, ClientId,
```

Comment:
`mpsc` and `proto` is not a part of parent module. it happens to because parent module include `tokio::mpsc` and `mqtt3::proto` in its scope. 
It should be organized in this manner everywhere in the code instead of `super`
```rust
use tokio::mpsc;
use mqtt3::proto;
use crate::{...}
```

Reply:
Make sense, fixed that.